JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt

```diff
@@ -50,9 +53,13 @@ fun PostCardTop(post: Post) {
     val typography = MaterialTheme.typography()
     Column(modifier = LayoutWidth.Fill + LayoutPadding(16.dp)) {
         post.image?.let { image ->
-            Container(modifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill) {
+            val sizeModifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill
```

Comment:
Ditto, previous comment. Why referenced twice?

Reply:
Same, workaround for dev06

Reply:
Done!